src/FileSystem/FileSystem.json

```diff
@@ -475,6 +475,50 @@
             "optional": true
           }
         }
+      },
+      {
+        "id": "CreateDirOptions",
```

Comment:
So strange that reference to `CreateDirOptions` and `RemoveDirOptions` did not surfaced any errors until APIs were actually used.